debug/customization/vector-tile-layer/smk-init.js

```diff
@@ -0,0 +1,298 @@
+                    place: {
+                        weight: 1,
+                        fillColor: '#f20e93',
+                        color: '#f20e93',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    housenumber: {
+                        weight: 1,
+                        fillColor: '#ef4c8b',
+                        color: '#ef4c8b',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    poi: {
+                        weight: 1,
+                        fillColor: '#3bb50a',
+                        color: '#3bb50a',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    },
+                    earth: {
+                        fill: true,
+                        weight: 1,
+                        fillColor: '#c0c0c0',
+                        color: '#c0c0c0',
+                        fillOpacity: 0.2,
+                        opacity: 0.4
+                    }
+                };
+
+                // Monkey-patch some properties for nextzen layer names, because
+                // instead of "building" the data layer is called "buildings" and so on
+                vectorTileStyling.buildings  = vectorTileStyling.building;
+                vectorTileStyling.boundaries = vectorTileStyling.boundary;
+                vectorTileStyling.places     = vectorTileStyling.place;
+                vectorTileStyling.pois       = vectorTileStyling.poi;
+                vectorTileStyling.roads      = vectorTileStyling.road;
+
+            	// Nextzen tiles PBF layer setup
+
+                // Assumes layers = "all", and format = "mvt"
+                var nextzenTilesUrl = "https://tile.nextzen.org/tilezen/vector/v1/512/all/{z}/{x}/{y}.mvt?api_key={apikey}";
+
+                var nextzenVectorTileOptions = {
+                    rendererFactory: L.canvas.tile,
+                    attribution: '<a href="https://nextzen.com/">&copy; NextZen</a>, <a href="http://www.openstreetmap.org/copyright">&copy; OpenStreetMap</a> contributors',
+                    vectorTileLayerStyles: vectorTileStyling,
+                    apikey: 'gCZXZglvRQa6sB2z7JzL1w',
+                };
+
+                var nextzenTilesPbfLayer = L.vectorGrid.protobuf(nextzenTilesUrl, nextzenVectorTileOptions);
+
+            	// ESRI tiles PBF layer setup
+                var esriStyle = {};
+                esriStyle.Continent  = vectorTileStyling.earth;
+                esriStyle.Bathymetry = vectorTileStyling.water;
+                esriStyle["Vegetation small scale"] = vectorTileStyling.landuse;
+                esriStyle["Marine area"] = vectorTileStyling.water;
+                esriStyle.Land = vectorTileStyling.earth;
+
+                esriStyle["City small scale"             ] = vectorTileStyling.building;
+                esriStyle["Admin0 point"                 ] = [];
```

Comment:
Does this actually need to be defined as an empty array?

Reply:
It looks to be useful. For example, if I remove the empty arrays for "water line" style names, rivers are given a thick blue style, which looks distracting.

Reply:
Could you add a code comment at the top of the declaration of esriStyle that explains what the empty arrays do? ie. They remove the styling for that object/proprty in the vector tile so they don't appear on the map.

Reply:
Done.